tests/test_entry_resource.py

```diff
@@ -12,7 +14,26 @@ def test_content_type(self, response):
         assert response.headers['content-type'] == 'application/json'
 
     @pytest.mark.xfail
-    def test_content_fieldnames(self, response):
-        assert 'entry-number' in response.json(), \
-            '''Missing required field `entry-number`
-            Ref: http://openregister.github.io/specification/#assert-df4a8358'''
+    def test_response_contents(self, response):
+        #This schema should always represent the response json specified at <http://openregister.github.io/specification/#entry-resource>
+        entry_schema = {
+            "type": "object",
+            "properties": {
+                "entry-number": {
+                    "type": "string",
+                    "pattern": "^\d+$"
+                },
+                "item-hash": {
+                    "type": "string",
+                    "pattern": "^sha-256:[a-z\d]+$"
```

Comment:
this is either too liberal (for hex strings) or too strict (for base64) and I can't remember what we decided on